auth/integration_test/src/integration_test.cc

```diff
@@ -101,6 +101,10 @@ class FirebaseAuthTest : public FirebaseTest {
   // Delete the current user if it's currently signed in.
   void DeleteUser();
```

Comment:
I guess technically this is supposed to be a utility function to facilitate tests, not an actual test of the API surface. We should have a proper TestDeleteUser for that, and a TestDeleteUser_DEPRECATED.

However, thinking about it, let's leave this, since deprecated users should *only* be using deprecated APIs, and thus DeleteUser_DEPRECATED should be used by the deprecated tests. Might be worth a TODO to add a TestDeleteUser in the future.